corim/meta.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2021 Contributors to the Veraison project.
```

Comment:
Perhaps we should clarify somewhere the use of word meta, is aligned with the CDDL schema and is used to carry information about the CoRIM Signer Entity?

Reply:
to that purpose I have expanded the description of the `Meta` type -- see 9277b88

Reply:
yes, looks ok to me now!